packages/server/package.json

```diff
@@ -57,32 +57,30 @@
   },
   "exports": {
     ".": {
-      "module": {
-        "browser": "./dist/emotion-server.browser.esm.js",
-        "default": "./dist/emotion-server.esm.js"
+      "types": {
+        "import": "./dist/emotion-server.cjs.mjs",
+        "default": "./dist/emotion-server.cjs.js"
       },
+      "module": "./dist/emotion-server.esm.js",
       "import": "./dist/emotion-server.cjs.mjs",
       "default": "./dist/emotion-server.cjs.js"
     },
     "./create-instance": {
-      "module": {
-        "browser": "./create-instance/dist/emotion-server-create-instance.browser.esm.js",
-        "default": "./create-instance/dist/emotion-server-create-instance.esm.js"
+      "types": {
+        "import": "./create-instance/dist/emotion-server-create-instance.cjs.mjs",
+        "default": "./create-instance/dist/emotion-server-create-instance.cjs.js"
       },
+      "module": "./create-instance/dist/emotion-server-create-instance.esm.js",
       "import": "./create-instance/dist/emotion-server-create-instance.cjs.mjs",
       "default": "./create-instance/dist/emotion-server-create-instance.cjs.js"
     },
     "./package.json": "./package.json"
   },
+  "imports": {},
   "preconstruct": {
     "entrypoints": [
       "./index.js",
       "./create-instance/index.js"
-    ],
-    "exports": {
-      "envConditions": [
-        "browser"
-      ]
-    }
```

Comment:
some packages had `browser` conditions but I didn't keep them because, from what I can tell, they were unused. I only kept them when I could recognize them as being used